internal/core/user/common/interface.go

```diff
@@ -14,11 +14,10 @@
 
 package common
 
-import "github.com/erda-project/erda/apistructs"
+import (
+	"github.com/erda-project/erda-proto-go/core/user/pb"
+)
 
 type Interface interface {
```

Comment:
I found you used `pb.UserServiceServer` directly in all related places, not `common.Interface`. So maybe we can delete this unused Interface.

Reply:
It is used as a wrapper for initial.
 in internal/core/user/provider.go